go/generate-grpcwebproxy-binaries.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+rm -rf build
+mkdir -p build
+
+GOOS=linux GOARCH=amd64 go build -o build/grpcwebproxy-v0.8.1-linux-amd64 ./grpcwebproxy
```

Comment:
Probably easier to inline this script in the release so you can use the tag name without passing bash arguments around.

Reply:
Inline it into `publish-release.sh`? Ah actually yes, I see I'm hard coding the version string - doh!